src/main/resources/static/codegen/built-api-spec.yml

```diff
@@ -7520,6 +7520,11 @@ components:
           items:
             type: string
             format: uuid
+        endDate:
```

Comment:
as discussed can we please change name (and description) to reflect that this applies to the default bed, not the room itself